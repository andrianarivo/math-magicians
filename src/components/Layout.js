import { Outlet } from 'react-router-dom';
import Navbar from './NavBar';

export default function Layout() {
  return (
    <div className="wrapper">
      <Navbar />
      <Outlet />
    </div>
  );
}
