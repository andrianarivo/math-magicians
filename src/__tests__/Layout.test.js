import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Layout from '../components/Layout';

describe('testing Layout component', () => {
  test('render', () => {
    const tree = renderer
      .create(
        <Router>
          <Layout />
        </Router>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
