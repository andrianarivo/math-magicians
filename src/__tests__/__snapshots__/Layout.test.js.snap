// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Layout component render 1`] = `
<div
  className="wrapper"
>
  <nav
    className="nav"
  >
    <h2>
      Math Magicians
    </h2>
    <ul
      className="ul"
    >
      <li
        className="li"
      >
        <a
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="li"
      >
        <a
          className=""
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li
        className="li"
      >
        <a
          className=""
          href="/quote"
          onClick={[Function]}
        >
          Quotes
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
